make/ci.mk

```diff
@@ -79,6 +80,10 @@ e2e.build.rhel-8.6: rhel86
 
 e2e.build.rhel-8.6-offline: rhel86-offline infra.aws.destroy
 
+e2e.build.rocky-9.1-offline: rocky91-offline infra.aws.destroy
+
+e2e.build.rocky-9.1: rocky91
+
 e2e.build.rhel-8.6-nvidia: rhel86-nvidia
 
```

Comment:
should we not do these any more?

Reply:
I am keeping it in sync with other related targets for other OS. We can remove all related target at once when we deprecate them in separate PR.